oktoberfest/utils/config.py

```diff
@@ -143,6 +143,23 @@ def search_results(self) -> Path:
     def search_results_type(self) -> str:
         """Get search type (Maxquant, Msfragger, Mascot or Internal) from the config file."""
         return self.inputs.get("search_results_type", "maxquant").lower()
+    
+
+    @property
+    def custom_modifications(self) -> dict:
```

Comment:
you need a proper type annotation here, it should be Dict[str, Dict[str, Tuple[str, float]]]